config/routes.rb

```diff
@@ -103,8 +103,8 @@
     resources :unsubscribe_feedbacks, only: %i[new create], controller: "jobseekers/unsubscribe_feedbacks"
   end
 
-  get "sign-up-for-NQT-job-alerts", to: "nqt_job_alerts#new", as: "nqt_job_alerts"
-  post "sign-up-for-NQT-job-alerts", to: "nqt_job_alerts#create", as: "new_nqt_job_alert"
+  get "sign-up-for-NQT-job-alerts", to: "subscriptions#new", as: "nqt_job_alerts", defaults: { origin: "/sign-up-for-NQT-job-alerts" }
```

Comment:
Rather than adding the origin param can we add `nqt_suitable` to the job role parameters?

Reply:
Not too sure what you mean. What job role parameters?

Reply:
The job role part of the search criteria parameter